.github/workflows/build-binaries.yml

```diff
@@ -40,11 +40,11 @@ jobs:
         run: |
           ./node_modules/@oclif/dev-cli/bin/run pack -t="${{ matrix.package-targets }}"
           mv ./dist/version ./dist/${{ matrix.os }}-version
-          mv ./dist/coveo-v${{ github.ref }}/coveo-v${{ github.ref }}.tar.gz ./dist/coveo-v${{ github.ref }}/${{ matrix.os }}-coveo-v${{ github.ref }}.tar.gz
+          mv ./dist/coveo-${{GITHUB_REF/refs\/tags\//}}/coveo-${{GITHUB_REF/refs\/tags\//}}.tar.gz ./dist/coveo-${{GITHUB_REF/refs\/tags\//}}/${{ matrix.os }}-coveo-${{GITHUB_REF/refs\/tags\//}}.tar.gz
```

Comment:
🤔  Seems like a cross branch pollution ?

Reply:
It's my fault on this one, I was trying to make the build work and thought that my merge message was bad and tried to change it.
I just fixed that in this PR